src/components/RacePage.js

```diff
@@ -133,66 +104,65 @@ export function RacePage() {
               />
             ))}
           </ul>
-          <h3 className="heading-3">Starting Grid</h3>
-            <ul>
-              {startingGrid
-                .sort((a, b) => a.position - b.position)
-                .map((gridPosition, index) => (
-                  <li key={index}>
-                    Position: {gridPosition.position}, Driver: {driversDetails[gridPosition.driver_number]}
-                  </li>
-                ))}
-            </ul>
+
+          <h3 className="heading-6 mb-16">Starting Grid</h3>
+          <ul className="flex flex-col w-fit m-auto">
+            {startingGrid
+              .sort((a, b) => a.position - b.position)
+              .map((gridPosition, index) => (
+                <li 
+                  key={index} 
+                  className="text-center w-fit even:-mt-32 even:ml-64 even:mb-24"
+                >
+                  <div className="text-sm font-display text-stone-500">P{gridPosition.position}</div>
+                  <div className="border-x-2 border-t-2 border-solid border-stone-500 px-4 pt-4 w-52">
+                    {driversDetails[gridPosition.driver_number]}
+                  </div>
+                </li>
+              ))}
+          </ul>
         </div>
 
+        <div className="sm:grow-0">
+          <div className="canvas-wrapper mb-64">
+            <ThreeCanvas imageFile="/maps/map.png" locData = {locData}/>
+          </div>
 
-        <div className="race-page__col grow">
-        <h3 className="heading-3">3D Track Visualization</h3>
-          <ThreeCanvas imageFile="/maps/map.png" locData = {locData}/>
-          <h3 className="heading-3">Lap Data</h3>
-          <p>View the lap duration for each driver below:</p>
-          <LineChart
-            width={1000}
-            height={300}
-            data={chartData}
-            margin={{ top: 5, right: 30, left: 20, bottom: 5 }}
-          >
-            <CartesianGrid strokeDasharray="3 3" />
-            <XAxis dataKey="name" />
-            <YAxis domain={[minYAxisValue, maxYAxisValue]} />
-            <Tooltip />
-            <Legend />
-            {[...new Set(laps.map(lap => lap.driver_acronym))].map((acronym, index) => (
-              <Line key={index} type="monotone" dataKey={acronym} stroke={`#${Math.floor(Math.random() * 16777215).toString(16)}`} />
-            ))}
-          </LineChart>
+          <h3 className="heading-6 mb-16">Lap Data</h3>
+          <LapChart laps={laps} setLaps={() => setLaps} driversDetails={driversDetails} className="lap-chart" />
         
-      
+          <h3 className="heading-6 mb-16">Tire Strategy</h3>
+          {/* TireStrategy component needs help see notes inside cmponent and uncomment linen below to see test version */}
+          {/* <TireStrategy drivers={drivers} /> */}
```

Comment:
this isn't super necessary because we pretty much already have this built. i was just playing around with it since we already use recharts for for laps chart.